Source/Core/VideoBackends/CMakeLists.txt

```diff
@@ -1,7 +1,12 @@
 add_subdirectory(OGL)
 add_subdirectory(Null)
 add_subdirectory(Software)
+
+if(CMAKE_SYSTEM_NAME STREQUAL "Windows")
```

Comment:
WIN32 is old style. New style should check against the actual name.
There isn't a similar variable for example for FreeBSD, so it's better to make them all similar.